doc/faq.md

```diff
@@ -134,11 +138,11 @@ It may need to write scripts to parse the outputs or integrated with your local
 
 <a id="support" />
 
-### Where can I found the support ? if doc not helps.
+### Where can I find the support if doc not helps?
 
-If any question, pls make sure you have read the `doc/` first, and `LVS`'s [documents](https://github.com/alibaba/LVS/tree/master/docs) are also helpful. It's the better to have some experience about networking configuration (e.g., routing, neigbour, ...) since DPVS is kernel-bypass, basic routing, IP address configurations are needed to be setup from scratch.
+If any question, pls make sure you have read the [docs](../doc) first, and `LVS`'s [documents](https://github.com/alibaba/LVS/tree/master/docs) are also helpful. It's better to have some experiences about networking configuration (e.g., routing, neigbour, ...) since DPVS is kernel-bypass, basic routing, IP address configurations are needed to be setup from scratch.
 
-We have Chinese `QQ-Group` or `WeChat-Group`, you can ask questions, raise issues, talk about design, help others or discuss everything else about DPVS. Here's the [entry](https://github.com/iqiyi/dpvs/issues/147) for WeChar-Group (微信群) and [entry](https://github.com/iqiyi/dpvs/issues/83) for QQ-Group (QQ群). Email to `qlb-devel #at# qiyi #dot# com` is another way.
+We have Chinese `QQ-Group` or `WeChat-Group`, you can ask questions, raise issues, talk about design, help others or discuss everything else about DPVS. Here's the [entry](https://github.com/iqiyi/dpvs/issues/147) for WeChar-Group (微信群) and [entry](https://github.com/iqiyi/dpvs/issues/83) for QQ-Group (QQ群). Email to `iig_cloud_qlb #at# qiyi #dot# com` is another way.
```

Comment:
for **WeChat-Group**.
**Emailing** to `iig_cloud_qlb #at# qiyi #dot# com` is another way **to contact us**.